cogs/utils/rsstools.py

```diff
@@ -6,7 +6,26 @@
 from __main__ import botdata
 from bs4 import BeautifulSoup
 
-    
+def is_new_blog(entry):
+    """Takes the newest dota blog entry, and checks data against record
+    returns a boolean
+    updates blog entry if it is new"""
+    new = parser.parse(entry.published) #date on 'new' entry
+    old = botdata.__getitem__("dotablog")
```

Comment:
You should be able to do `old = botdata["dotablog"]` for this

Reply:
Ah, that would make sense. Will make these fixes as well.